examples/Cimpress.Extensions.Http.Caching.Redis.Examples/Cimpress.Extensions.Http.Caching.Redis.Examples.csproj

```diff
@@ -0,0 +1,36 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Examples to use Cimpress.Extensions.Http.Caching.Redis</Description>
+    <AssemblyTitle>Examples to use Cimpress.Extensions.Http.Caching.Redis</AssemblyTitle>
+    <VersionPrefix>0.0.1</VersionPrefix>
+    <Authors>Markus Thurner;Amanda Lobel</Authors>
+    <TargetFramework>netcoreapp1.5</TargetFramework>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <DebugType>portable</DebugType>
+    <AssemblyName>Cimpress.Extensions.Http.Caching.Redis.Examples</AssemblyName>
+    <PackageId>Cimpress.Extensions.Http.Caching.Redis.Examples</PackageId>
+    <GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>
+    <PackageTargetFallback>$(PackageTargetFallback);dotnet5.6;dnxcore50;portable-net45+win8;dotnet</PackageTargetFallback>
+    <RuntimeFrameworkVersion>1.1.1</RuntimeFrameworkVersion>
+    <GenerateAssemblyConfigurationAttribute>false</GenerateAssemblyConfigurationAttribute>
+    <GenerateAssemblyCompanyAttribute>false</GenerateAssemblyCompanyAttribute>
+    <GenerateAssemblyProductAttribute>false</GenerateAssemblyProductAttribute>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\src\Cimpress.Extensions.Http.Caching.Redis\Cimpress.Extensions.Http.Caching.Redis.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.3.0" />
+    <PackageReference Include="System.ComponentModel.TypeConverter" Version="4.3.0" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.3.0" />
+    <PackageReference Include="FluentAssertions" Version="4.19.4" />
+    <PackageReference Include="xunit" Version="2.3.0" />
+    <PackageReference Include="moq" Version="4.6.38-alpha" />
```

Comment:
This should be `<PackageReference Include="moq.netcore" Version="4.4.0-beta8" />`